src/components/MeasurementsToolList.vue

```diff
@@ -33,7 +36,10 @@ const tools = computed(() => {
 });
 
 const remove = (id: ToolID) => {
-  props.toolStore.removeTool(id);
+  const imageID = currentImageID.value;
+  if (!imageID) return;
```

Comment:
No imageID, but showing remove button in toolList would be strange.  Throw Error or derive imageID from toolID so imageID is guarrentied?  Worried that defensive programming could lead to corner case runtime quirks.

Reply:
I like deriving the image ID from the tool ID in this case.